app/models/product.rb

```diff
@@ -37,12 +38,50 @@ class Product < ApplicationRecord
   has_one_attached :image
   has_many :simplo_items
   has_one :store
+  has_one :stock, dependent: :destroy
 
   with_options presence: true do
     validates :name
     validates :price, numericality: { greater_than: 0 }
```

Comment:
Some parent products have prices equal to zero. They might be needed in future, update this validation.